tests/decode_images.rs

```diff
@@ -391,6 +391,10 @@ fn oom() {
 
     match err {
         tiff::TiffError::LimitsExceeded => {}
```

Comment:
This line should be removed if the error below is now the one our decoder returns for this fuzzer generated test case.